### PR TITLE
Presentation slide notes

### LINKED ISSUE #46: Upload Presentation Notes: QA Lead
**Describe what needs to get done**

Upload slides that pertain to your role. Markdown files for each topic are located in the `reports` directory. 

**Describe the solution you'd like**

Each team member should upload their contents into the `presentation_slides` branch and submit a PR.

### PR DESCRIPTION
**Pull request by edouarde1**
closes #46

### CODE REVIEW

File: docs/reports/requirements/2_dfd-desc.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Data Flow Diagrams
 
-Section 2 of the Requirements Report
 
-An overview of your system architecture presented as a data flow diagram (DFD), at both levels 0 and level 1, along with accompanying descriptions for each diagram. You must be very specific in this DFD so to design it in a way that supports an incremental and iterative development process. You can read an intro on DFDs and adopt its notation. Remember that a process does something, so be sure to use a verb to label your processes. In describing your DFD, highlight clearly which components will be delivered for each of the following milestones: peer testing #1, peer testing #2, and the final product. You may additionally wish to include extra components and label them as bonus features that may be developed if time allows.
+## Level 0 DFD 
 
-Diagrams will be incorporated into the PDF report.
+![](dfd_0.png)
```
Comment (vnajack) [inline docs/reports/requirements/2_dfd-desc.md:6]: I recommend adding in the point-form description for each level (basically what you said during the presentation). Then, we can use the point-form list this weekend when we write up the report!

**Review by vnajack — APPROVED** [review]
Looks great! just the one recommendation

File: docs/reports/requirements/2_dfd-desc.md (hunk 1 above)
Comment (edouarde1) [reply docs/reports/requirements/2_dfd-desc.md:6]: Thanks for the heads up! I have made those changes :)

File: docs/reports/requirements/6_test-strat.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Test Strategy
 
-Section 6 of the Requirements Report
+## Test Strategy - GitHub Actions
```
Comment (LoganParker) [inline docs/reports/requirements/6_test-strat.md:3]: Let me know if you'd like help setting this up - we should work together on this :)

**Review by LoganParker — APPROVED** [review]
This looks awesome! thanks for adding your research :)

**Pull request merged**